README.md

```diff
@@ -37,7 +37,7 @@ source 'https://github.com/CocoaPods/Specs'
 platform :ios, '9.0'
 
 target 'TARGET_NAME' do
-    pod 'TwilioVideo', '~> 2.0'
+    pod 'TwilioVideo', '~> 3.0.0'
```

Comment:
Would it be possible to allow any minor version? Twilio Video uses semVer.

```suggestion
    pod 'TwilioVideo', '~> 3.0'
```

Reply:
Ah, good point!